index.html

```diff
@@ -10,36 +10,210 @@
-             
-             <div class="social">
-                <p class="contact">LET’S CONNECT</p>
-                <ul class="social-connection">
-                    <li><a href="https://twitter.com/HenaryKhuraijam"><img class="social-icon"src="images/twiter.png" alt="twiter"></a></li>
-                    <li><a href="https://www.linkedin.com/in/henary-khuraijam-50487317a/"><img class="social-icon"src="images/Linkedin icon.png" alt="Linkedin"></a></li>
-                    <li><a href="#"><img class="social-icon"src="images/M.png" alt="M"></a></li>
-                    <li><a href="https://github.com/HenaryKhuraijam"><img class="social-icon"src="images/github.png" alt="github"></a></li>
-                    <li><a href="#"><img class="social-icon"src="images/V.png" alt="V"></a></li>
-                </ul>
-            </div>
-                   
-        </section>        
-
-    </main>
+      </header>
+   <main class="wrapper" id="main-container">
+    <section class="flex" id="headlines">
+      <div class="content-placeholder">
+        <h1 class="head-1" id="main-title">
+          I’m Henary,<br>
+          Glad to see you!
+        </h1>
+        <p class="description" id="title-desc">
+          I’m a software developer! I can help you build a product, 
+          feature or website Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, d
+          on’t hestiate to contact me.
+        </p>
+        <div class="social-media">
+          <h3 class="head3" id="social-1">LET’S CONNECT</h3>
+          <ul class="inline-li">
+            <li class="social-items">
+              <a href="https://twitter.com/HenaryKhuraijam">
+                <img src="images/twitter.png" alt="twitter link">
+              </a>
+            </li>
+            <li class="social-items">
+              <a href="https://www.linkedin.com/in/henary-khuraijam-50487317a/">
+                <img src="images/linkedin.png" alt="linkedin link">
+              </a>
+            </li>
+            <li class="social-items">
+              <a href="https://medium.com/">
+                <img src="images/medium.png" alt="medium link">
+              </a>
+            </li>
+            <li class="social-items">
+              <a href="https://github.com/HenaryKhuraijam">
+                <img src="images/github.png" alt="github link">
+              </a>
+            </li>
+            <li class="social-items">
+              <a href="https://angel.co/">
+                <img src="images/angellist.png" alt="angellist link">
+              </a>
+            </li>
+          </ul>
+        </div> 
+      </div>   
+    </section>
+    <section class="works" id="works">
```

Comment:
- [ ] You have done a commendable job implementing the desired requirements 💯, but kindly make sure you apply the right background color to your Works section.
-   Your design:
    ![henry1](https://user-images.githubusercontent.com/87394385/212494968-a26188c7-feda-439d-94da-a684b8232f23.JPG)

-   Figma design:
    ![henry2](https://user-images.githubusercontent.com/87394385/212494998-f9bb47a8-1f24-4216-9540-f60155652d3d.JPG)

-   Please check the background color(on the areas I have marked) in the design, and apply it to your design.